File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_user_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusUserVO:
    """System User Value Object"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    uuid: str = None
    face_image: ImageMediaResource = None
    face_video: ImageMediaResource = None
    username: str = None
    nickname: str = None
    email: str = None
    phone: str = None
    gender: str = None
    status: str = None
    role_ids: List[int] = None
    role_names: List[str] = None
    social_info_list: SocialInfoList = None

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusUserVO.kt
package com.sdkwork.backend

data class PlusUserVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val id: Int? = null,
    val uuid: String? = null,
    val faceImage: ImageMediaResource? = null,
    val faceVideo: ImageMediaResource? = null,
    val username: String? = null,
    val nickname: String? = null,
    val email: String? = null,
    val phone: String? = null,
    val gender: String? = null,
    val status: String? = null,
    val roleIds: List<Int>? = null,
    val roleNames: List<String>? = null,
    val socialInfoList: SocialInfoList? = null
)
